js/exercise-00.js

```diff
@@ -0,0 +1,38 @@
+/* 
+   Create a constant variable of each of the following primitive types:
+   - String
+   - Number
+   - Boolean
+   - Null
+   - Undefined
+
+   Log out each variable to the terminal at the end.
+*/
+
+//Declare variables
+
+        const stringVariable = "Hello cats!";
```

Comment:
Unnecessary spaces.

```suggestion
const stringVariable = "Hello cats!";
```

FIX: remove initial spaces